Confus/Confus/Player.cpp

```diff
@@ -44,6 +44,8 @@ namespace Confus
         createAudioEmitter();
         startWalking();
 
+		m_PlayerHealth.setHealth(100);
```

Comment:
If this is what we use sethealth for, wouldn't it be better if we were able to pass it to the constructor of 'Health'?